framework/include/iotbus/iotbus_gpio.h

```diff
@@ -91,6 +91,7 @@ typedef void (*gpio_isr_cb)(void *user_data);
 /**
  * @brief initializes gpio_context based on gpio pin.
  *
+ * @details @b #include <iotbus_gpio.h>
```

Comment:
This is wrong.
```
#include <iotbus/iotbus_gpio.h>
```
Isn't it?

Reply:
I did not fully understand the doxygen usage. I'll fix it.